packages/react-components/src/components/Select/Select.css

```diff
@@ -17,6 +17,13 @@
   color: var(--typography-color-disabled);
 }
 
+/* Rext description below select input */
```

Comment:
Typo `Rext`

Reply:
This is fixed in [9f3973c](https://github.com/bcgov/design-system/pull/437/commits/9f3973cf94aabf261f4181b43e577c374bd12556)